tests/ustring/asc.bas

```diff
@@ -0,0 +1,144 @@
+#include "fbcunit.bi"
+#include "ustring.bi"
+
+
+#macro hCheckString( U, w )
+  r = 0                                               'reset flag
+
+  CU_ASSERT_EQUAL(u, w)
```

Comment:
This is duplicated in many (all?) of the tests.  Could be an include file instead.  Seems like the macro should generate the PASS/FAIL, rather than setting a flag.